### PR TITLE
TIC-77: Create select season and edit season page

### PR DESCRIPTION
**Pull request by HaliTran**
### Description
Added the Seasons Ticketing Dashboard route. Please note, I left comments in the files that I actually changed, the other are just prettier formatting changes.

### Risks
No risks, simply adding the route and dashboard link so admin can visit seasons page. Currently doesn't show anything.

### Validation
Able to see it within ticketing manager UI

### Issue
https://pdx-hkaus.atlassian.net/jira/software/projects/TIC/boards/1?selectedIssue=TIC-77

### Operating System
M2 Mac

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -31,65 +31,119 @@ import Doorlistmain from './components/Ticketing/ticketingmanager/Doorlistpage/d
...
+        />
+        <Route
+          path='/ticketing/addnewsletter'
+          element={<ProtectedRoute component={NewsletterCreatemain} />}
+        />
+        <Route
+          path='/ticketing/showings'
+          element={<ProtectedRoute component={Showingsmain} />}
+        />
+        <Route
```
Comment (HaliTran) [inline client/src/App.tsx:131]: I added this route, everything else in the file is prettier format change

File: client/src/components/Ticketing/ticketingmanager/Season/SeasonMain.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
Comment (HaliTran) [inline client/src/components/Ticketing/ticketingmanager/Season/SeasonMain.tsx:1]: Added this file

File: client/src/components/Ticketing/ticketingmanager/dashboard.tsx [hunk 3]
```diff
@@ -10,152 +10,173 @@ import DashboardLink from '../userdashboard/DashboardLink';
...
+                  d='M8 7V3m8 4V3m-9 8h10M5 21h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v12a2 2 0 002 2z'
                 />
               </svg>
-            )}
-            title="Manage Seasonal Tickets"
-            route="/ticketing/seasonaltickets"
+            }
+            title='Seasons'
+            route='/ticketing/seasons'
           />
```
Comment (HaliTran) [inline client/src/components/Ticketing/ticketingmanager/dashboard.tsx:119]: Added this DashboardLink, the rest of the page is prettier

File: client/src/App.tsx [hunk 4]
```diff
@@ -31,65 +31,119 @@ import Doorlistmain from './components/Ticketing/ticketingmanager/Doorlistpage/d
...
+          path='/ticketing/seasons'
+          element={<ProtectedRoute component={SeasonsMain} />}
+        />
```
Comment (HaliTran) [inline client/src/App.tsx:134]: I added this route, everything else in the file is prettier format change

File: client/src/components/Ticketing/ticketingmanager/Season/SeasonMain.tsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+/**
+ * Main page to host seasons page
+ *
+ * @module
+ * @returns Seasons Page
+ */
+
+const SeasonsMain = () => {
+  return (
+    <div className='flex flex-row'>
```
Comment (henrykaus) [inline client/src/components/Ticketing/ticketingmanager/Season/SeasonMain.tsx:14]: I know this is really tiny, but I think `flex-row` is the default flex-direction so its inclusion is redundant.

**Review by henrykaus — COMMENTED** [review]
I think everything looks pretty good. I think since seasons are a bit of a big deal, it would make sense to place that button as the fourth button (where Create Newsletter is now), rather than as the fifth smaller button

File: client/src/components/Ticketing/ticketingmanager/Season/SeasonMain.tsx (hunk 5 above)
Comment (HaliTran) [reply client/src/components/Ticketing/ticketingmanager/Season/SeasonMain.tsx:14]: Fixed!

**Review by henrykaus — APPROVED** [review]
LGTM

**Pull request merged**